crates/editor/src/git.rs

```diff
@@ -90,8 +90,8 @@ pub fn diff_hunk_to_display(
         let hunk_end_row = hunk.row_range.end.max(hunk.row_range.start);
         let hunk_end_point = Point::new(hunk_end_row.0, 0);
 
-        let multi_buffer_start = snapshot.buffer_snapshot.anchor_after(hunk_start_point);
-        let multi_buffer_end = snapshot.buffer_snapshot.anchor_before(hunk_end_point);
+        let multi_buffer_start = snapshot.buffer_snapshot.anchor_before(hunk_start_point);
```

Comment:
We ran into an issue where when comparing the start points between the diff hunks and the expanded hunks the biases would be different.

Adjusting this line seemed to fix the problem.

@SomeoneToIgnore Do you know if the previous biasing was intentional/load-bearing?

Reply:
I vaguely recall taking this pattern from some place, but the old code indeed looks like a mistake to me now.